bin/full_conversion.py

```diff
@@ -0,0 +1,49 @@
+
+import argparse
+import ROOT
+
+def main():
+    parser = argparse.ArgumentParser(description='Convert detector')
+    parser.add_argument('-c', '--compact', help='Compact file location(s)',
+                        required=True, type=str, nargs='+')
+    parser.add_argument('-o', '--out', help='Converted file path',
+                        default='nothing', type=str)
+    parser.add_argument('-v', '--visibility', help='Level of layers in detector',
+                        default=9, type=int)
+
+    args = parser.parse_args()
+
+    convert(args.compact, args.out, args.visibility)
+
+
+def convert(compact_files, out_path, visibility):
```

Comment:
This seems to be the same function as in `conversion_xml2root.py` so we could also just import it from there instead of copying it over.